src/lib/array.jl

```diff
@@ -197,6 +197,26 @@ for (mapfunc,∇mapfunc) in [(:map,:∇map),(:pmap,:∇pmap),(:vmap,:∇vmap)]
   end
 end
 
+function ∇pmap(cx, f, wp, args...; kwargs...)
+  ys_and_backs = pmap((args...) -> _pullback(cx, f, args...), wp, args...; kwargs...)
+  if isempty(ys_and_backs)
+    ys_and_backs, _ -> nothing
+  else
+    ys, backs = unzip(ys_and_backs)
+    ys, function (Δ)
+      # Apply pullbacks in reverse order. Needed for correctness if `f` is stateful.
+      Δf_and_args_zipped = pmap((f, δ) -> f(δ), wp, _tryreverse(pmap, backs, Δ)...)
```

Comment:
`_tryreverse` isn't doing anything, right? Nor is the comment true, I think.

Reply:
Yeah the comment isn't true on pmap because it needs to be pure anyways because it distributes.